### DISCUSSION TITLE
Change in docker ee support in 09/2022

### DISCUSSION THREAD

**Discussion by uBpringlIoNaRys** [actions/virtual-environments#5592] [General questions]
As announced here: https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment?tabs=Windows-Server#install-docker Microsoft will not provide a Docker EE build anymore and support will be moved to Mirantis.

Which container runtime will the microsoft hosted agents migrate to?

**Comment by miketimofeev** [comment]
Hi @uBpringlIoNaRys!
Thanks for raising this. We will investigate the possible options and come back with the results

**Reply by uBpringlIoNaRys** [reply]
Is there an update to this?

**Reply by miketimofeev** [reply]
@uBpringlIoNaRys yes, we have made an announcement about switching to Docker CE
https://github.com/actions/runner-images/issues/6181
